itests/src/test/resources/testconfiguration.properties

```diff
@@ -6,6 +6,7 @@ minimr.query.files=\
 
 # Queries ran by both MiniLlapLocal and MiniTez
 minitez.query.files.shared=\
+  dynpart_sort_optimization_distribute_by.q,\
```

Comment:
do we need to run this test with minitez - or it may run with minillaplocal?

Reply:
For some reason, The issue is not reproducible with LLAP, Hence running this with mini tez